addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1334,77 +1430,90 @@ var SnippetsMenu = Widget.extend({
         }, false);
     },
     /**
-     * Disable the overlay editor of the active snippet and activate the new one
-     * if given.
+     * Disable the overlay editor of the active snippet and activate the new one.
      * Note 1: if the snippet editor associated to the given snippet is not
      *         created yet, this method will create it.
      * Note 2: if the given DOM element is not a snippet (no editor option), the
      *         first parent which is one is used instead.
      *
-     * @param {jQuery|false} $snippet
+     * @private
+     * @param {jQuery|false} $snippetBlock
      *        The DOM element whose editor (and its parent ones) need to be
      *        enabled. Only disable the current one if false is given.
      * @param {boolean} [previewMode=false]
      * @param {boolean} [ifInactiveOptions=false]
      * @returns {Promise<SnippetEditor>}
      *          (might be async when an editor must be created)
      */
-    _activateSnippet: async function ($snippet, previewMode, ifInactiveOptions) {
+    _activateSnippet: async function ($snippetBlock, previewMode, ifInactiveOptions) {
         if (this._blockPreviewOverlays && previewMode) {
             return;
         }
-        if ($snippet && !$snippet.is(':visible')) {
+        if (!$snippetBlock.is(':visible')) {
```

Comment:
No it doesn't seem like it can. So indeed the doc is behind and will be updated.
Now I also see that for some reason we don't seem to have an exact replacement for that. We instead use `_enableLastEditor` which disables all snippet editors, then enables the "last snippet editor". I'll look into that on Monday.

Reply:
@qsm-odoo Since we got our Monday hijacked by AL/FP and @Zinston does not work on Wednesdays, we will assume he meant Thursday, right ? :p

Reply:
Yesterday was mostly highjacked too but I finally got it done and it fixes the popover snippet issue.
So I was wrong to say we don't have an exact replacement for that, `_enableLastEditor` does cover that case actually as `_activateSnippet(false)` in fact _does_ try to reactivate the last snippet. So the issue was that some lines were missing for some reason, that a boolean was wrongly set, and that we were not deactivating the option (`toggleOptions(false)`) when deactivating. All this is related to the `previewMode` variable not being there in `_enableLastEditor` and that bit of missing code.
https://github.com/odoo/odoo/pull/61030/files

Reply:
Now I'm going to look into the `previewMode === true` case so it will fix the issue mentioned above with the todo.